### ISSUE TITLE
The integration does not detect the switch on the local network

### ISSUE THREAD

**Issue by ksanya48** [slydiman/sscpoe#7]
The integration does not detect the switch on the local network
PS308G
![Screenshot_2024-11-15-12-54-50-09_46d7dcdee588e16f072522d110a15d46](https://github.com/user-attachments/assets/01672987-6a85-4273-8793-e45ce5a85f9b)
![Screenshot_2024-11-15-12-59-57-42_c3a231c25ed346e59462e84656a70e50.jpg](https://github.com/user-attachments/assets/d27ec6ed-1421-4eeb-820d-6b415797dba1)

![Screenshot_2024-11-15-12-56-06-78_21da60175e70af211acc4f26191b7a77.jpg](https://github.com/user-attachments/assets/56be5791-b5ed-4c4d-b7bf-dae890af7f63)

**Comment by ksanya48** [comment]
![Uploading Screenshot_2024-11-15-12-59-57-42_c3a231c25ed346e59462e84656a70e50.jpg…]()

**Comment by ksanya48** [comment]
2024-11-15 13:21:59.296 DEBUG (SyncWorker_7) [custom_components.sscpoe] SSCPOE_local_send: {'cmd': 'calludp', 'syn': '3AHwGLnH', 'data': {'callcmd': 'search'}}
2024-11-15 13:22:01.319 DEBUG (SyncWorker_7) [custom_components.sscpoe] SSCPOE_local_recv: Timeout
2024-11-15 13:22:03.090 INFO (MainThread) [habluetooth.scanner] hci0 (68:4E:05:4C:E2:70): Bluetooth scanner has gone quiet for 90s, restarting
2024-11-15 13:22:03.322 DEBUG (SyncWorker_7) [custom_components.sscpoe] SSCPOE_local_recv: Timeout

**Comment by slydiman** [comment]
Try to cancel and select `Add SSCPOE device` again. The integration may detect the local device on the 2nd or 3rd try.

Download [Windows desktop config app](http://www.sscee.com/en/en/col.jsp?id=105) and check that it detects PS308G locally.

Note the protocol uses multicast UDP. If your HA server has 2 or more IPs (multiple network interfaces) it may be reason of this issue.

I see the device version 6.0.240630. I have tested PS308G and GPS204 with the firmware version 6.0.231024. Probably the developer changed something. First confirm that `Windows desktop config app` works. Then I need Wireshark traffic records during the searching procedure from `Windows desktop config app` to investigate what has been changed in the protocol.

**Comment by ksanya48** [comment]
I have tried to update the device many times (15-20 times). Today I will not be able to connect through the Windows application (there is no way)...Previously, I used local integration, but I needed to reinstall it... and after that, I can't set it up... I deleted it in hacs, reinstalled different versions, but there is no result

**Comment by slydiman** [comment]
Try to reboot the POE switch (power off/on).
If you cannot connect locally to the device using SSC Windows app, it means something is wrong with the device. Try reset it to factory settings via the smartphone app.

**Comment by alexander-saratovcev** [comment]
In case you using iptables of ufw you need to allow multicast. 
For example: 

`sudo ufw allow proto udp to 224.0.0.0/4`

**Comment by Megachip** [comment]
> Try to cancel and select `Add SSCPOE device` again. The integration may detect the local device on the 2nd or 3rd try.

Same here. Web interface is reachable. Any chance to enter IP manually? (Tested on unprovisioned, provisioned and cloud connected)

**Comment by slydiman** [comment]
> Same here. Web interface is reachable.

Web interface of what? SSC devices with firmware v6.x have no any web interface. 
If your SSC device is older, most probably it uses different local protocol. 
Currently SSC devices use multicast UDP for local communications. Multicast UDP works only in the same subnet.

**Comment by Megachip** [comment]
@slydiman how can we help to fix the not detecting issue?

**Comment by slydiman** [comment]
> how can we help to fix the not detecting issue?

Did you try Windows app sscpoe? Can it detect the switch locally in the same network?

**Comment by Megachip** [comment]
> > how can we help to fix the not detecting issue?
> 
> Did you try Windows app sscpoe? Can it detect the switch locally in the same network?

Only the mobile ones, which working fine.

**Comment by slydiman** [comment]
Try the Windows sscpoe app first. If it does not work, then something is wrong with your network.
If the Windows app works, we can try to investigate the optimal TTL or other multicast UDP parameters.

**Comment by jcpflp** [comment]
You are right: I needed to switch off "IGMP snooping" on the router. Thanks.
